packages/contracts-ops/contracts/CallBatch.sol

```diff
@@ -161,6 +160,27 @@ abstract contract CallBatch is Script {
         kvs[1][1] = vm.toString(data);
         writeSimpleObject("", "", kvs, true);
     }
+
+    function prank(address router) public {
```

Comment:
i would maybe consider `prankExecuteBatch()` or something like that since `CallBatch` will be in the inheritance tree, it's a little clearer what's happening as opposed to `vm.prank`

Reply:
agreed 👍

Reply:
https://github.com/nomad-xyz/monorepo/pull/507/commits/e7e8a5ad769985a70d6629c7e6561ccaa470f864